src/components/panels/details/info-multiple-selection.tsx

```diff
@@ -51,16 +70,140 @@ export default function InfoMultipleSection() {
+
+    return (
+        <>
+            <Box>
+                <Heading as="h5" size="sm">
+                    {t('panel.details.multipleSelection.selected')} {selected.size}
+                </Heading>
+                <VStack m="var(--chakra-space-1)">
+                    <HStack m="var(--chakra-space-1)" width="100%">
+                        <Button
+                            size="sm"
+                            width="100%"
+                            variant={filterNodes ? 'solid' : 'ghost'}
+                            onClick={() => setFilterNodes(!filterNodes)}
+                            colorScheme="blue"
+                        >
+                            {t('panel.details.multipleSelection.filterNodes')}
+                        </Button>
+                        <Button
+                            size="sm"
+                            width="100%"
+                            variant={filterEdges ? 'solid' : 'ghost'}
+                            onClick={() => setFilterEdges(!filterEdges)}
+                            colorScheme="blue"
+                        >
+                            {t('panel.details.multipleSelection.filterEdges')}
+                        </Button>
+                    </HStack>
+                    {(filterEdges || filterNodes) && (
+                        <Button
+                            m="var(--chakra-space-1)"
+                            width="100%"
+                            size="sm"
+                            onClick={() => setIsOpenChangeModal(true)}
+                            variant="outline"
+                            colorScheme="blue"
+                        >
+                            {t('panel.details.multipleSelection.change')}
+                        </Button>
+                    )}
+                    {[...selected]
+                        .filter(id => filterNodes || !(id.startsWith('stn') || id.startsWith('misc_node')))
+                        .filter(id => filterEdges || !id.startsWith('line'))
+                        .map(id => (
+                            <HStack key={id} width="100%">
+                                <Button
+                                    width="100%"
+                                    size="sm"
+                                    variant="solid"
+                                    onClick={() => dispatch(setSelected(new Set([id])))}
+                                    overflow="hidden"
+                                    maxW="270"
+                                    textOverflow="ellipsis"
+                                    whiteSpace="nowrap"
+                                    display="ruby"
+                                >
+                                    {getName(id)?.replaceAll('\\', '⏎')}
+                                </Button>
+                                <Button size="sm" onClick={() => dispatch(removeSelected(id))}>
```

Comment:
Bonus if we could make a multi-selection checkbox and remove them in one click :)

Reply:
Also some users might be confused whether this _X_ is removing the current from the selected or from the canvas. How about having another deletion button?